docs/threads/introduction.md

```diff
@@ -4,87 +4,89 @@ hero_img: /images/threads-hero.png
 
-The first three concepts a developer will encounter with ThreadsDB are [Databases](#databases), [Collections](#collections), and [Instances](#instances). The organization is simple. Instances are the individual records you create, update, or delete. Instances are stored in a Collection. Collections have one or many Schemas and can only store Instances that match one of those Schemas. Databases can store many Collections.
+The first three concepts a developer will encounter with ThreadDB are [Databases](#databases), [Collections](#collections), and [Instances](#instances). The organization is simple. Instances are the individual records you create, update, or delete. Instances are stored in a Collection. Collections have one or many Schemas and can only store Instances that match one of those Schemas. Databases can store many Collections.
 
-ThreadsDB supports Mongodb/Mongoose style search. In the JavaScript library, you might write queries like the following.
+A Thread-based Database is tied to a single Thread (with associated Thread ID).
 
-```js
-Players.find(
-  { 
-    $and: [
-      { points: { $gt: 10 } },
-      { points: { $lt: 20 } },
-      { team: 'Astronauts' }
-    ]
-  }, 
-  { sort: { points: -1 }}
-)
+
+```typescript
+import { Database, JSONSchema, FilterQuery } from '@textile/threads-database'
+import { ThreadID } from '@textile/threads-id'
 ```
 
-### Multi-user Databases
+To start a new, empty database is simple. A new [Level Datastore](https://github.com/ipfs/js-datastore-level) is used as the backing store by default if no datastore is explicitly supplied. See the doc-strings for the `Database.constructor` for further options.
 
-Everything above just looks like a Database, so what's a Thread? ThreadsDB combines the storage and management of data (the Database) with networking, access control, and replication over IPFS using the Threads Protocol. The Threads protocol has been extensively documented in the [whitepaper](https://docsend.com/view/gu3ywqi), but in short, Threads use private-key encryption to manage both security and identity among multiple parties that can access or edit the same Database.
+By default, a ThreadDB will connect with a local `go-threads` Threads Daemon. See the [`go-threads`](https://github.com/textileio/go-threads) for details on getting started with local development. The Threads Daemon may be helpful to developers that aim to build their own Thread Services, host replication services, or test advanced Thread usage. The Threads Daemon (`threadsd`) is provided as an installable binary with every [release of Threads](https://github.com/textileio/go-threads/releases).
 
-Jump to [Thread Networking](#networking) to read more.
+Alternatively, it is possible to connect with a remote daemon by specifying the networking component of the Database ([See more](#replication-with-the-hub) for details on connecting to hosted remote services).
```

Comment:
yep, maybe we flip this and say, alternatively it's possible to connect to a local

Reply:
We should, but at the moment that isn't true, and it might not work very well yet. Can we get this in first and change it, or should we wait for those updates to js-threads?

Reply:
iterations for the win